### PR TITLE
remove only return true function

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: test/sol6/kyberNetwork.js [hunk 1]
```diff
@@ -2074,53 +2073,6 @@ contract('KyberNetwork', function(accounts) {
         });
     });
 
-    describe("test handle change edge case", async function(){
```
Comment (ilanDoron) [inline test/sol6/kyberNetwork.js:2077]: why these tests removed?

Comment (ductm54) [reply test/sol6/kyberNetwork.js:2302]: no need to mock to test when required is delete

**Comment by ilanDoron** [comment]
But these tests are still important.

Seems they test we handle this edge case correctly correctly
Don't follow why it should be removed.

File: contracts/sol6/Dao/KyberDAO.sol [hunk 2]
```diff
@@ -437,10 +437,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
 
         hasClaimedReward[staker][epoch] = true;
         // call fee handler to claim reward
-        require(
-            feeHandler.claimStakerReward(staker, perInPrecision, epoch),
-            "claimReward: feeHandle failed to claim"
-        );
+        feeHandler.claimStakerReward(staker, perInPrecision, epoch);
```
Comment (manhlx3006) [inline contracts/sol6/Dao/KyberDAO.sol:440]: May be we can change FeeHandler to:
`require(success, "staker rewards transfer failed");`
---> ` if (!success) { return false; }` and emit event

Comment (ilanDoron) [reply contracts/sol6/Dao/KyberDAO.sol:440]: for readability?

File: test/sol6/kyberDao.js [hunk 3]
```diff
@@ -3586,35 +3585,6 @@ contract('KyberDAO', function(accounts) {
       await feeHandler.withdrawAllETH({from: accounts[0]});
     });
 
-    it("Test claim reward should revert fee handler return false for claimReward", async function() {
```
Comment (manhlx3006) [inline test/sol6/kyberDao.js:3589]: Should still keep this test, but change to: "Test claim reward should revert if fee handler failed to claim reward` and make it revert in FeeHandler `claimStakerReward` function

File: contracts/sol6/Dao/KyberDAO.sol (hunk 2 above)
Comment (manhlx3006) [reply contracts/sol6/Dao/KyberDAO.sol:440]: Yeah mostly for readability. Not sure if we should catch this event and notify staker that this address can not receive ETH

Comment (ilanDoron) [reply contracts/sol6/Dao/KyberDAO.sol:440]: so idea is to add an event in feeHander.

Comment (Anyhowclick) [reply contracts/sol6/Dao/KyberDAO.sol:440]: IMO leave it like that.. feeHandler should revert if the failing msg
